salt/cni/policy/default-kube-dex-policy.yaml.jinja

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: kube-dex-policy
```

Comment:
Can we set the names to something like this:

    name: suse:caasp:dex-networkpolicy

Lots of our resources now use this pattern, so it's blindingly obvious these are created+managed by CaaSP etc.

Reply:
yes I will modify that

Reply:
worker-0:~ # kubectl apply -f  dns.yaml
The NetworkPolicy "suse:caasp:dns-networkpolicy" is invalid: metadata.name: Invalid value: "suse:caasp:dns-networkpolicy": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')

Reply:
Arg. Some resources get validated with a DNS name pattern. I'm not sure I understand why K8S would apply that to NetworkPolicy resources though. The same happened with PodSecurityPolicy resources.

For those, we used: 

    metadata:
      name: suse.caasp.psp.privileged

Can we do the same here and use periods instead of colons.